CODE_OF_CONDUCT.md

```diff
@@ -21,10 +21,11 @@ Examples of unacceptable behavior by participants include:
 * Public or private harassment
 * Publishing others' private information, such as a physical or electronic address, without explicit permission
 * Other conduct which could reasonably be considered inappropriate in a professional setting
+* Use of unacceptable commit text or username is also prohibited.
 
 ## Our Responsibilities
 
-Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instance of unacceptable behavior.
```

Comment:
@RahulSharma099  I applaud your attention to detail for the typo found in the "Code of Conduct" Markdown file.  Changed noted along Line #28 in the revised file.

📓  Learning how to communicate in the proper manner on GitHub.